src/locale/mr.js

```diff
@@ -52,19 +89,19 @@ export default moment.defineLocale('mr', {
         sameElse : 'L'
     },
     relativeTime : {
-        future : '%s नंतर',
-        past : '%s पूर्वी',
-        s : 'सेकंद',
-        m: 'एक मिनिट',
-        mm: '%d मिनिटे',
-        h : 'एक तास',
-        hh : '%d तास',
-        d : 'एक दिवस',
-        dd : '%d दिवस',
-        M : 'एक महिना',
-        MM : '%d महिने',
-        y : 'एक वर्ष',
-        yy : '%d वर्षे'
+        future: '%sमध्ये',
```

Comment:
That was my first thought too. But then if you see following translations are not correct (in case of future time).
```
काही सेकंदांत
%d मिनिटांत
%d तासांत
%d दिवसांत
%d महिन्यांत
%d वर्षांत
```
The correct translations would be (अनुस्वाराशिवाय)
```
काही सेकंदात
%d मिनिटात
%d तासात
%d दिवसात
%d महिन्यात
%d वर्षात
```
But in the case of past time, correct translation would be (अनुस्वारासह)
```
काही सेकंदांपूर्वी
%d मिनिटांपूर्वी
%d तासांपूर्वी
%d दिवसांपूर्वी
%d महिन्यांपूर्वी
%d वर्षांपूर्वी
```
So in that case we will need that extra check of `isFuture` to decide whether we should use अनुस्वार or not. Instead if we use `'%sमध्ये'` we can avoid that extra check and uniformly use the translation with अनुस्वार. That was the reason I chose `'%sमध्ये'` instead of `'%sत'`. Hope that makes sense.